clients/http/command_test.go

```diff
@@ -50,6 +50,18 @@ func TestIssueGetCommandByName(t *testing.T) {
 	require.IsType(t, &responses.EventResponse{}, res)
 }
 
+func TestIssueGetCommandByNameWithQueryParams(t *testing.T) {
+	deviceName := "Simple-Device01"
+	cmdName := "SwitchButton"
+	path := path.Join(common.ApiDeviceRoute, common.Name, deviceName, cmdName)
+	ts := newTestServer(http.MethodGet, path, &responses.EventResponse{})
+	defer ts.Close()
+	client := NewCommandClient(ts.URL)
+	res, err := client.IssueGetCommandByNameWithQueryParams(context.Background(), deviceName, cmdName, map[string]string{"foo": "bar"})
+	require.NoError(t, err)
```

Comment:
Can you add code to verify the query params where actually in the resulting REST request.

Reply:
@hahattan  we have the default values in the core-command, so we should just add the mock function to behave as the core-command default values here

Reply:
Need to get this merged ASAP, once TSC approved after code Freeze